.circleci/config.yml

```diff
@@ -24,12 +31,80 @@ jobs:
           # don't limit this to the number of allocated cores, the job is
           # likely to get OOMed and killed.
           GOOPTS: "-p 2"
+  publish:
+    executor: golang
+    steps:
+    - checkout
+    - setup_remote_docker:
+        version: 18.06.0-ce
+    - run:
+        name: Setup environment variables
+        command: |
+          echo "export DOCKER_IMAGE_TAG=`date +%Y%m%d`" >> $BASH_ENV
+          echo "export DOCKER_ORG=${DOCKER_ORG:-prombench}" >> $BASH_ENV
+    - prometheus/publish_images:
+        container_image_name: prombench
+        dockerfile_path: "prombench/cmd/prombench/Dockerfile"
+        registry: docker.io
+        organization: "$DOCKER_ORG"
+        login_variable: DOCKER_LOGIN
+        password_variable: DOCKER_PASSWORD
+    - prometheus/publish_images:
+        container_image_name: funcbench
+        dockerfile_path: "funcbench/Dockerfile"
+        registry: docker.io
+        organization: "$DOCKER_ORG"
+        login_variable: DOCKER_LOGIN
+        password_variable: DOCKER_PASSWORD
+    - prometheus/publish_images:
+        container_image_name: amgithubnotifier
+        dockerfile_path: "tools/amGithubNotifier/Dockerfile"
+        registry: docker.io
+        organization: "$DOCKER_ORG"
+        login_variable: DOCKER_LOGIN
+        password_variable: DOCKER_PASSWORD
+    - prometheus/publish_images:
+        container_image_name: comment-monitor
+        dockerfile_path: "tools/commentMonitor/Dockerfile"
+        registry: docker.io
+        organization: "$DOCKER_ORG"
+        login_variable: DOCKER_LOGIN
+        password_variable: DOCKER_PASSWORD
+    - prometheus/publish_images:
+        container_image_name: fake-webserver
+        dockerfile_path: "tools/fake-webserver/Dockerfile"
+        registry: docker.io
+        organization: "$DOCKER_ORG"
+        login_variable: DOCKER_LOGIN
+        password_variable: DOCKER_PASSWORD
+    - prometheus/publish_images:
+        container_image_name: scaler
+        dockerfile_path: "tools/scaler/Dockerfile"
+        registry: docker.io
+        organization: "$DOCKER_ORG"
+        login_variable: DOCKER_LOGIN
+        password_variable: DOCKER_PASSWORD
+    - prometheus/publish_images:
+        container_image_name: load-generator
+        dockerfile_path: "tools/load-generator/Dockerfile"
+        registry: docker.io
+        organization: "$DOCKER_ORG"
+        login_variable: DOCKER_LOGIN
+        password_variable: DOCKER_PASSWORD
 
 workflows:
   version: 2
   prombench:
     jobs:
-    - test:
+    - build_and_test:
         filters:
-          tags:
+          branches:
             only: /.*/
+    - publish:
+        requires:
+          - build_and_test
+        context: testinfra
```

Comment:
@simonpasquier do you know any predefined context with credentials of prombench@docker.io ?

Reply:
some hints: the image created here should be pushed under `docker.io/prombench` instead of `docker.io/prom`, context variable DOCKER_LOGIN / DOCKER_PASSWORD should be set accordingly.